dist/rpm.dockerfile

```diff
@@ -35,27 +35,20 @@ case "${OS_VERSION}" in
         chmod a+x /etc/scl_enable
         ;;
     8|9)
-        yum install -y epel-release rubygems ruby-devel
-        ;;
-    *)
-        if [ "${OS_NAME}" = "fedora" ]; then
-            yum install -y rubygems ruby-devel
-        else
-            yum install -y epel-release rubygems ruby-devel
-        fi
+        yum install -y rubygems ruby-devel glibc-langpack-en glibc-locale-source
         ;;
 esac
 
 # Disable postgres
-if command -v dnf; then dnf -qy module disable postgresql; fi
```

Comment:
what was it and why we don't need it? 😅

Reply:
I had the same question

Reply:
Sorry this was a stray change which I made when fiddling with the rhel 8 build. I've reverted it.